okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java

```diff
@@ -58,4 +66,27 @@ public String getMessage() {
   public void setMessage(String message) {
     this.message = message;
   }
+
+  public Stage getStage() {
+    return stage;
+  }
+
+  public void setStage(Stage stage) {
+    this.stage = stage;
+  }
+
+  /**
+   * Clone an entry with only original tenant module information (Before async install).
+   * @return entry
+   */
+  @JsonIgnore
+  public TenantModuleDescriptor cloneLegacy() {
```

Comment:
Maybe just me, `Legacy` gives me an impression that this method will go away and should not be used. :)

Reply:
Fair. It's not going a way. I'll rename it again.